health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/AppConfigurationWrapper.js

```
@@ -0,0 +1,382 @@
+  const [showPopUp, setShowPopUp] = useState(false);
+  const searchParams = new URLSearchParams(location.search);
+  const fieldMasterName = searchParams.get("fieldType");
+  const module = "dummy-localisation";
+  const { mutateAsync: localisationMutate } = Digit.Hooks.campaign.useUpsertLocalisation(tenantId, module, "en_IN");
+  const { isLoading: isLoadingAppConfigMdmsData, data: AppConfigMdmsData } = Digit.Hooks.useCustomMDMS(
+    Digit.ULBService.getCurrentTenantId(),
+    MODULE_CONSTANTS,
+    [{ name: "AppScreenConfigTemplateSchema" }, { name: fieldMasterName }, { name: "DrawerPanelConfig" }, { name: "AppScreenLocalisationConfig" }],
+    {
+      cacheTime: Infinity,
+      staleTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)